src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -390,7 +390,8 @@ def createlist(self, request):
         for idx, row in enumerate(rows):
             item = FacilityListItem.objects.create(
                     row_index=idx,
-                    raw_data=','.join(row.raw_json.values()),
+                    raw_data=','.join(
+                        f'"{value}"' for value in row.raw_json.values()),
```

Comment:
could we just use **str(value)**?

Reply:
I thought about it, but no. To maintain backward compatibility, it's necessary to enclose the values in single quotes first, and then in double quotes. This will generate the following string: '"Name value","Street, value", "Country value"'. While generating the contributor fields, the code csv.reader(csv_data.split('\n'), delimiter=',') splits the raw_data based on commas between double quotes. This helps avoid a situation where the address is split into separate values.